deploy/README.md

```diff
@@ -0,0 +1,8 @@
+# Deploying the search client to a custom Wordpress page template
+
+1. Run `npm run build` to build this project.
+2. Figure out which Wordpress theme you're using and navigate to the directory for that theme under wp\_content.
+3. Add the code in ourroots\_cms\_functions.php to the end of the functions.php file in that directory. If you are upgrading a previous installation, you may want to change the version numbers from 0.0.1 to something else for cache-busting.
+4. Copy ourroots\_cms.php, main.css, and main.js from this directory to that directory.
+5. Copy the .js files from build/static/js to that directory, renaming main....chunk.js to main.chunk.js and renaming the other ...chunk.js file to other.chunk.js.
```

Comment:
I don't know how often we're going to do this, or how hard it is to rename, but if we do it frequently (or automated), could we write a simple script to do this rename?

For that matter, can we write a script to do the whole thing?

Reply:
Most people use filezilla or some other ftp client to manage their wordpress installations. I'm not sure a script would be that helpful.

Reply:
Got it. I thought these were instructions for us, the developers, not for end users. Looks good.